byoqm/source_coordinator/query_translations.py

```diff
@@ -0,0 +1,145 @@
+query_lang = {
```

Comment:
```suggestion
translate_to = {
```

Because then when its used in code we get syntax like

```py
(_ [{translate_to[self.coordinator.prog_lang]["parameters"]}] @parameters)
```